README.md

```diff
@@ -7,59 +7,45 @@
 [![Travis](https://travis-ci.org/openshift/source-to-image.svg?branch=master)](https://travis-ci.org/openshift/source-to-image)
 [![License](https://img.shields.io/github/license/openshift/source-to-image.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
 
-Source-to-Image (`S2I`) is a tool for building reproducible Docker images. `S2I` produces
-ready-to-run images by injecting source code into a Docker image and *assembling*
-a new Docker image for the application.  The result is then ready to use with `docker run`. 
+Source-to-Image (S2I) is a toolkit and workflow for building reproducible Docker images from source code. S2I produces
+ready-to-run images by injecting source code into a Docker container and letting the container prepare that source code for execution. By creating these self-assembling **builder images**, you can version and control your build environments exactly like you use Docker images to version your runtime environments.
```

Comment:
Confusing what "these builder images" refers to. It follows from the previous sentence which is talking about producing application images.